dist/images/ovnkube.sh

```diff
@@ -475,97 +515,154 @@ ovn-northd () {
 
 # v2 v3 - run ovnkube --master
 ovn-master () {
+  trap 'kill $(jobs -p); exit 0' TERM
   check_ovn_daemonset_version "2 3"
-  # this is only run on masters
-  # ovn-master - master node only
-  echo "=============== ovn-master (wait for ovs) ========== MASTER ONLY"
-  wait_for_ovs
+  rm -f /var/run/openvswitch/ovnkube-master.pid
```

Comment:
If we want to "fix" the pidfile race condition that Girish was talking about, then we could add the ovnkube.sh pid (from $BASHPID) to the filename of the ovnkube master pidfile (and others). That way subsequent invocations of the ovnkube.sh script wouldn't find stale pidfiles from older runs of the components.